Nicer yaml2json error reporting

```
Traceback (most recent call last):
  File "./yaml2json.py", line 3, in <module>
    json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=2, separators=(',', ': '))
  File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 93, in safe_load
    return load(stream, SafeLoader)
  File "/usr/lib/python2.7/dist-packages/yaml/__init__.py", line 71, in load
    return loader.get_single_data()
  File "/usr/lib/python2.7/dist-packages/yaml/constructor.py", line 37, in get_single_data
    node = self.get_single_node()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 82, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 111, in compose_sequence_node
    node.value.append(self.compose_node(node, index))
  File "/usr/lib/python2.7/dist-packages/yaml/composer.py", line 77, in compose_node
    "second occurence", event.start_mark)
yaml.composer.ComposerError: found duplicate anchor 'way_area'; first occurence
  in "<stdin>", line 27, column 5
second occurence
  in "<stdin>", line 162, column 5
make: *** [cleartables.json] Error 1
```
We can catch `yaml.composer.ComposerError` give a nicer message then exit 1. This avoids an uninformative backtrace

--- a/yaml2json.py
+++ b/yaml2json.py
@@ -1,3 +1,7 @@
 #!/usr/bin/env python
 import sys, yaml, json
-json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=2, separators=(',', ': '))
+try:
+    json.dump(yaml.safe_load(sys.stdin), sys.stdout, indent=2, separators=(',', ': '))
+except yaml.YAMLError, e:
+    sys.stderr.write("YAML error:\n%s\n" % e)
+    sys.exit(1)

